EventFilter/OnlineMetaDataRawToDigi/test/unpackMetaData.py

```diff
@@ -0,0 +1,50 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('TEST')
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+
+process.source = cms.Source(
+    "NewEventStreamFileReader",
+    fileNames = cms.untracked.vstring(
+        "file:/afs/cern.ch/user/m/mommsen/scratch0/run302163_ls0787_streamExpress_StorageManager.dat"
```

Comment:
private files and files on /afs better be avoided in the release.
Please put it in /store/user